winui/Ribbon/Ribbon-QAT.md

```diff
@@ -121,6 +121,20 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
 
 ![WinUI Ribbon QAT menu items](Ribbon-QAT-images/winui-ribbon-qat-menu-items.png)
 
+## Add items to QAT customize dialog window
+
+To open QAT Customize Window, select `MoreCommands` option from the ContextMenu of the QAT. In the QAT Customized dialog Window, the list of Commands is available. The Commands can be filtered only from the Particular tab by using `Choose commands from` option. Then, select the Command to add to the QuickAccessToolBar and add commands to the right Pane of the Quick Access ToolBar Dialog by clicking Add Button. Finally click OK.
+
```

Comment:
To open QAT customize dialog window, select `MoreCommands` option from the ContextMenu of the QAT. This dialog window showcases all available commands within the Ribbon control. 

By using `Choose commands from` option, you can filter the command based on ribbon tabs. Then, select the desired command and click `Add` button to add the command to the right Pane of the QAT dialog window. Finally, click `OK`.

Reply:
modified